data/abdqudus.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Abdqudus Gbadamosi",
+  "bio": "Open Source Enthusiast| Frontend Developer",
+  "socials": [
+    {
+      "icon": "FaLinkedin",
+      "url": "http://linkedin.com/in/abdqudus-gbadamosi",
+    },
+    {
+      "icon": "FaTwitter",
+      "url": "http://twitter.com/abdkhudus",
```

```suggestion
      "url": "http://twitter.com/abdkhudus"
```
The trailing comma isnot required